Mathlib/Topology/Baire/CompleteMetrizable.lean

```diff
@@ -71,15 +71,15 @@ instance (priority := 100) BaireSpace.of_pseudoEMetricSpace_completeSpace : Bair
   let r : ℕ → ℝ≥0∞ := fun n => (F n).2
   have rpos : ∀ n, 0 < r n := by
     intro n
-    induction' n with n hn
-    exact lt_min εpos (Bpos 0)
-    exact Hpos n (c n) (r n) hn.ne'
+    induction n with
```

Comment:
do you really need `induction`, or is `cases` enough here?

Reply:
We really do, yeah.